### DISCUSSION TITLE
https for rest api in swagger ? [Apr 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1944] [Discord Threads]
> 🙋‍♂️ **Zorphix**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Did you mean to use that in swagger ? if so you can use customUrl in dropdown  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/967488966817374288)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by GuodongFan** [reply]
hi, what's the meaning of "Autogenerated from discord"?
Is this thread coming from discord?

**Reply by wingkwong** [reply]
@GuodongFan Yes. We move threads from [Discord](https://discord.gg/5RgZmkW) to Discussions from time to time.
